uktrade/data-hub-frontend
# Add the TabNav component

## Description of change

Added the `TabNav` component which is a stateful wrapper of [`govuk-react/tabs`](https://govuk-react.github.io/govuk-react/?path=/story/tabs--component-default).

## Test instructions

There's nothing to see as the component is not used anywhere yet. It will be covered with functional tests once it is used on the homepage to switch between _my companies lists_ and _my referrals_.
